.github/CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ For more nuanced testing runs, check out more detailed documentation [here](http
 ## Creating [Pull Requests](https://github.com/capitalone/DataProfiler/pulls)
 Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
 
-1. Fork the repo and create your branch from `main`.
+1. Fork the repo and create your branch from `dev`.
```

Comment:
Fixed this after a note from a contributor --> all PRs should be into `dev`. PR to `main` only when releasing